RecipesLyricsMashup/RecipesLyricsMashupV2/RecipeLyricsMashup/RecipeLyricsMashup/Pages/Index.cshtml.cs

```diff
@@ -19,23 +19,26 @@ public IndexModel(ILogger<IndexModel> logger)
         {
             _logger = logger;
         }
+        [BindProperty(SupportsGet = true)]
+        public string SearchString { get; set; } //Morgan added
 
         public IActionResult OnGet(string search)
         {
             Encoding ascii = Encoding.ASCII;
             //string search = HttpUtility.HtmlEncode(sort);
-            Console.WriteLine(search);
+            //Console.WriteLine(search);

```

Comment:
commented this line out becuase it is not needed.

Reply:
Remove this entirely.  It's considered bad practice to commit commented-out code to version control, because you can always go back in history to see what used to be there.  You are in your own branch, you can do anything you want without impacting Master.